makefiles/arch/native.inc.mk

```diff
@@ -0,0 +1,224 @@
+NATIVEINCLUDES += -DNATIVE_INCLUDES
+NATIVEINCLUDES += -I$(RIOTBASE)/core/lib/include/
+NATIVEINCLUDES += -I$(RIOTBASE)/core/include/
+NATIVEINCLUDES += -I$(RIOTBASE)/sys/include/
+NATIVEINCLUDES += -I$(RIOTBASE)/drivers/include/
+
+ifeq ($(OS),Darwin)
+  DEBUGGER ?= lldb
+else
+  DEBUGGER ?= gdb
+endif
+
+# only use pyterm wrapper if term target is requested
+ifeq (,$(filter term,$(MAKECMDGOALS)))
+  RIOT_TERMINAL ?= native
+else
+  RIOT_TERMINAL ?= pyterm
+endif
+
+RESET ?= $(RIOTBOARD)/native/dist/reset.sh
+FLASHER ?=
+FLASHFILE ?= $(ELFFILE)
+
+ifeq (pyterm,$(RIOT_TERMINAL))
+  TERMFLAGS += -ps $(FLASHFILE)
+  PROCARGS := --process-args
+else
+  TERMPROG ?= $(FLASHFILE)
+endif
+
+ZEP_IP ?= [::1]
+ZEP_PORT_BASE ?= 17754
+ifeq (1,$(USE_ZEP))
+ifeq (pyterm,$(RIOT_TERMINAL))
+  TERMFLAGS += --process-args '-z $(ZEP_IP):$(ZEP_PORT_BASE)'
+else
+  TERMFLAGS += -z $(ZEP_IP):$(ZEP_PORT_BASE)
+endif
+endif
+ifneq (,$(ZEP_MAC))
+ifeq (pyterm,$(RIOT_TERMINAL))
+  TERMFLAGS += --process-args '\-\-eui64=$(ZEP_MAC)'
+else
+  TERMFLAGS += --eui64=$(ZEP_MAC)
+endif
+endif
```

Comment:
Better keep those 'board' specific things to `boards/native/common_Makefile.include` this file should only about toolchain specific things.